### PR TITLE
Chapter 1 Patches

### PR DESCRIPTION
**Pull request by dhurlburtusa**
Here are some suggested changes for chapter 1. Please let me know if there is anything you are not fond of.

You can leave comments on any of the files changed by going to the "Files changed" tab.

You can also look at the commits tab to see each individual commit and the associated message. The message may be helpful in understanding the reason for the change.

### CODE REVIEW

File: graphicstext/c01-introduction/s3.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@ you a bit of background about the history of OpenGL and about the graphics
 hardware that supports it.</p1>
 
 <p>In the first desktop computers, the contents of the screen were managed
-directly by the <word>CPU</word>.  For example, to draw a line segment on the screen, the CPU
+directly by the <newword>CPU</newword>.  For example, to draw a line segment on the screen, the CPU
```
Comment (dhurlburtusa) [inline graphicstext/c01-introduction/s3.xml]: I cloned the repo and searched for uses of CPU, API, and vertex. I believe these are the first time they are used in the book.

File: graphicstext/c01-introduction/s3.xml [hunk 2]
```diff
@@ -41,7 +41,7 @@ GPU can understand.</p>
 <p>OpenGL is not the only graphics API.  The best-known alternatives are probably
 the proprietary APIs used by Apple and Microsoft.  Apple's MacOS uses an API known
 as Metal, while Microsoft Windows uses Direct3D.  Nevertheless, OpenGL
-can currently still be used on MacOS and Windows.  Furthermore, a newer open API named
+can still be used on MacOS and Windows.  Furthermore, a newer open API named
```
Comment (dhurlburtusa) [inline graphicstext/c01-introduction/s3.xml:44]: I think removing the word "currently" makes it read better.

File: graphicstext/c01-introduction/s3.xml (hunk 1 above)
Comment (davidjeck) [reply graphicstext/c01-introduction/s3.xml]: I'm probably not very consistent about how &lt;word> and &lt;newword> are used, but &lt;newword> is really meant for when a definition of the word is given in the text, so not exactly for the first occurrence of the word in the book.

Comment (dhurlburtusa) [reply graphicstext/c01-introduction/s3.xml]: I reverted this changes to using `<newword>`. For upcoming PRs, I won't make changes to these tags.

**Comment by dhurlburtusa** [comment]
@davidjeck I reverted some suggestions. Please let me know if you'd like anything else updated for chap 1. If not, then this PR is ready to be merged. Let me know if you need assistance in doing that.

BTW, I am working on chap 2. I'll roughly half way through. I'll try to finish today.

**Pull request merged**